tests/pkg/fixtures/cert_subject_mapping_queries.go

```diff
@@ -61,6 +64,9 @@ func FindCertSubjectMappingForApplicationTemplate(t *testing.T, ctx context.Cont
 
 		for _, csm := range currentCertSubjectMappings.Data {
 			if str.PtrStrToStr(csm.InternalConsumerID) == appTemplateID {
+				require.Contains(t, csm.Subject, expectedCN)
```

Comment:
why not equal? or function that compares subject properties in case they are in a different order?

Reply:
Because the subjects are in different format and I'm only interested if the matched csm by the internal consumer ID has the CN that I passed when creating the certificate.
These are the subject values:
csm.Subject: _CN=ord-svc-system-with-csm-discovery,OU=47b4575a-f102-414a-8398-2d973ad65f3a+OU=SAP Cloud Platform Clients+OU=Region,O=SAP SE,L=local,C=DE_
subject from certificate in the e2e test: _/C=DE/O=SAP SE/OU=SAP Cloud Platform Clients/OU=Region/OU=47b4575a-f102-414a-8398-2d973ad65f3a/L=local/CN=ord-svc-system-with-csm-discovery-test_